sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusModelFactory.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Messaging.ServiceBus.Primitives
+{
+    /// <summary>
+    /// This class contains methods to create certain ServiceBus models.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class ServiceBusModelFactory
+    {
+        /// <summary>
+        /// Creates a new ServiceBusReceivedMessage instance for mocking.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ServiceBusReceivedMessage ServiceBusReceivedMessage(
+            ServiceBusMessage sentMessage,
+            bool isSettled,
+            int deliveryCount,
+            DateTimeOffset lockedUntil,
+            long sequenceNumber,
+            string deadLetterSource,
+            short partitionId,
+            long enqueuedSequenceNumber,
+            DateTimeOffset enqueuedTime,
+            Guid lockTokenGuid,
+            object bodyObject
```

Comment:
This is internal, so we shouldn't need a param to set it.

Reply:
I've removed all parameters associated with internal only properties.